services/ordering/src/EShopOnAbp.OrderingService.Domain/Orders/Order.cs

```diff
@@ -57,6 +57,15 @@ internal Order SetOrderAccepted(Guid paymentRequestId, string paymentRequestStat
         return this;
     }
 
+    public Order SetOrderCancelled(Guid paymentRequestId, string paymentRequestStatus)
+    {
+        PaymentRequestId = paymentRequestId;
+        PaymentStatus = paymentRequestStatus;
+        OrderStatus = OrderStatus.Cancelled;
```

Comment:
### TLDR:
Cancel order, not paymentRequest. Argument should be **orderStatus**, not `paymentRequestStatus`.
### Long Version:
This is not correct.
Payment **needs to be completed** in order to **accept** the order. After order is accepted, if user (or admin application users) cancel the order, they should directly **cancel the the order**. 

PaymentStatus should only be changed **after PaymentService is refunded the order**, which we will handle in later milestones.